.env

```diff
@@ -43,4 +43,5 @@ MESSENGER_TRANSPORT_DSN=doctrine://default?auto_setup=0
 ###< symfony/mailer ###
 ARTICLES_ON_PAGE=10
 PROGRAMME_MIN_TIME_MINUTES=15
-PROGRAMME_MAX_TIME_MINUTES=360
\ No newline at end of file
+PROGRAMME_MAX_TIME_MINUTES=360
+EVOZON_WAREHOUSE_BASE_URI=''
```

Comment:
Why empty string? It should lead to the API

Reply:
Is empty in .env which is on github. Is not empty in .env.local wich is not on github... I think Rares told us we have this possibility too and I wanted to try it.